package.json

```diff
@@ -2,7 +2,7 @@
   "name": "dev.to",
   "repository": {
     "type": "git",
-    "url": "https://github.com/Kumo-by-Theodo/articles.git"
+    "url": "https://github.com/PChol22/kumo-articles.git"
```

Comment:
You will be able to move this article to Kumo later, but don't forget to do it. You can ask Fred for the invitation link.